network/core-node/admin-guide/running-node.mdx

```diff
@@ -3,178 +3,110 @@ title: Running
 
 ## Joining the Network
 
 Your node will go through the following phases as it joins the network:
 
-### Establishing Connection to Other Peers.
+### Establish Connection to Other Peers.
 
 You should see `authenticated_count` increase.
 
-<CodeExample>
-
 ```json
 "peers" : {
-   "authenticated_count" : 3,
-   "pending_count" : 4
+  "authenticated_count" : 3,
+  "pending_count" : 4
 },
 ```
 
-</CodeExample>
-
 ### Observing Consensus
 
 Until the node sees a quorum, it will say:
 
-<CodeExample>
-
 ```json
 "state" : "Joining SCP"
 ```
 
-</CodeExample>
-
-After observing consensus, a new field `quorum` will display information about network decisions. At this point the node will switch to "_Catching up_":
-
-<CodeExample>
+After observing consensus, a new field `quorum` will display information about network decisions. At this point the node will switch to a "_Catching up_" state:
 
 ```json
 "quorum" : {
-   "qset" : {
-      "ledger" : 22267866,
-      "agree" : 5,
-      "delayed" : 0,
-      "disagree" : 0,
-      "fail_at" : 3,
-      "hash" : "980a24",
-      "missing" : 0,
-      "phase" : "EXTERNALIZE"
-   },
-   "transitive" : {
-      "intersection" : true,
-      "last_check_ledger" : 22267866,
-      "node_count" : 21
-   }
+  "qset" : {
+    "ledger" : 22267866,
+    "cost" : 20883268,
+    "agree" : 5,
+    "delayed" : 0,
+    "disagree" : 0,
+    "fail_at" : 3,
+    "hash" : "980a24",
+    "lag_ms" : 430,
+    "missing" : 0,
+    "phase" : "EXTERNALIZE"
+  },
+  "transitive" : {
+    "intersection" : true,
+    "last_check_ledger" : 22267866,
+    "node_count" : 21
+  }
 },
 "state" : "Catching up",
 ```
 
-</CodeExample>
-
 ### Catching up
 
-This is a phase where the node downloads data from archives. The state will start with something like:
-
-<CodeExample>
+This is a phase where the node downloads data from any configured archives. This phase begins with something like:
 
 ```json
 "state" : "Catching up",
 "status" : [ "Catching up: Awaiting checkpoint (ETA: 35 seconds)" ]
 ```
 
-</CodeExample>
-
-And then go through the various phases of downloading and applying state such as
-
-<CodeExample>
+And will then move through the various phases of downloading and applying state, such as:
 
 ```json
 "state" : "Catching up",
 "status" : [ "Catching up: downloading ledger files 20094/119803 (16%)" ]
 ```
 
-</CodeExample>
+You can specify how far back in time your node goes to catch up in your config file. If you set the `CATCHUP_COMPLETE` field to `true`, your node will replay the _entire history_ of the network, which can take a long time. Weeks. Satoshipay offers a [parallel catchup script](https://github.com/satoshipay/stellar-core-parallel-catchup) to speed up the process, but you only need to replay the complete network history if you're setting up a Full Validator. Otherwise, you can specify a starting point for catchup using the `CATCHUP_RECENT` field. See the [complete example configuration] for more details.
```

Comment:
I think this section needs some validation.   First, I don't know that the Satoshipay parallel catchup has been maintained, so I think it's worth checking to make sure that's still a thing.  Second, I think we may want to suggest against CATCHUP_COMPLETE even for Full validators vs. suggesting they clone someone else's archive (since I'm sure replaying the network history from scratch will take a lot longer now, and may not even be possible)

Reply:
I did wonder if the satoshipay catchup project was still something. How does someone go about cloning an existing history archive?

Reply:
Yeah: the last commit to the Satoshipay repo is ~6 years ago.  So I believe that link should be removed.

I imagine that there's a way to just download an archive, but I'm not sure.  But more importantly: do we ever want people to run a complete catchup in their Stellar Core node?  If we don't, then let's change this to say so.  But I think someone else should way in because I'm not sure.